### PR TITLE
映射文档 No 20

### PR DESCRIPTION
**Pull request by hongtushi**
#5691 
这个参数out，我不太确定算不算tensor的参数

### CODE REVIEW

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.arcsinh.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## [ torch 参数更多 ]torch.Tensor.arcsinh
+
+### [torch.Tensor.arcsinh](https://pytorch.org/docs/stable/generated/torch.Tensor.arcsinh)
+
+```python
+torch.arcsinh(input, *, out=None)
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.arcsinh.md]: 这里分类应该是无参数，调用方式应该为 Tensor.arcsin()，参考 https://pytorch.org/docs/stable/generated/torch.Tensor.arcsin.html?highlight=arcsin#torch.Tensor.arcsin

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.arcsinh.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+```
+
+### [paddle.asinh](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/asinh_cn.html)
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.arcsinh.md]: 这里paddle 对应的api 为 `paddle.Tensor.asinh`;  调用方式为 Tensor.asinh(); 这个函数在官网好像没有写 https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/Tensor_cn.html，
但paddle.asinh() 和  Tensor.asinh() 是不一样的，前者是普通调用，后者是类方法的调用。

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.select.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+## [ torch 参数更多 ]torch.Tensor.select
+
+### [torch.Tensor.select](https://pytorch.org/docs/stable/generated/torch.Tensor.select)
+
+```python
+torch.select(input, dim, index)
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.select.md]: 这里的调用方式应该是 `Tensor.select(dim, index)`，参考链接 https://pytorch.org/docs/stable/generated/torch.Tensor.select.html?highlight=select#torch.Tensor.select

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.select.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+```
+
+### [paddle.index_select](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/index_select_cn.html)
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.select.md]: 这里的名称应该是 `Tensor.index_select(index, axis=0, name=None)`，参考链接 https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/Tensor_cn.html#index-select-index-axis-0-name-none

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.sigmoid.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+## [ torch 参数更多 ]torch.Tensor.sigmoid
+
+### [torch.Tensor.sigmoid](https://pytorch.org/docs/stable/generated/torch.Tensor.sigmoid)
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.sigmoid.md:3]: 这里的问题同上

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.sigmoid.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+```python
+torch.sigmoid(input, *, out=None)
+```
+
+### [paddle.nn.functional.sigmoid](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/functional/sigmoid_cn.html)
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.sigmoid.md]: 这里对应的 API 应该是 `paddle.Tensor.sigmoid`

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.sign.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
+## [ torch 参数更多 ]torch.Tensor.sign
+
+### [torch.Tensor.sign](https://pytorch.org/docs/stable/generated/torch.Tensor.sign)
+
+```python
+torch.sign(input, *, out=None)
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.sign.md]: 问题同上

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.sin.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
+## [ torch 参数更多 ]torch.Tensor.sin
+
+### [torch.Tensor.sin](https://pytorch.org/docs/stable/generated/torch.Tensor.sin)
+
+```python
+torch.sinh(input, *, out=None)
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.sin.md]: 问题同上

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.sinh.md [hunk 9]
```diff
@@ -0,0 +1,22 @@
+## [ torch 参数更多 ]torch.Tensor.sinh
+
+### [torch.Tensor.sinh](https://pytorch.org/docs/1.13/generated/torch.Tensor.sinh.html?highlight=torch+tensor+sinh#torch.Tensor.sinh)
+
+```python
+torch.sinh(input, *, out=None)
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.sinh.md]: 问题同上

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.slogdet.md [hunk 10]
```diff
@@ -0,0 +1,21 @@
+## [ 仅参数名不一致 ]torch.Tensor.slogdet
+
+### [torch.Tensor.slogdet](https://pytorch.org/docs/stable/generated/torch.Tensor.slogdet.html?highlight=torch+tensor+slogdet#torch.Tensor.slogdet)
+
+```python
+torch.slogdet(input)
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.slogdet.md]: 问题同上

**Review by Fancy-hjyp — COMMENTED** [review]
共性问题：

对于调用 `Tensor.* `方法是，其API 名称为 `torch.Tensor.*` ，调用方式为 `Tensor.*`，辛苦修改一下~

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.arcsinh.md [hunk 11]
```diff
@@ -0,0 +1,15 @@
+## [ 无参数 ]torch.Tensor.arcsinh
+
+### [torch.Tensor.arcsinh](https://pytorch.org/docs/stable/generated/torch.Tensor.arcsinh)
+
+```python
+torch.Tensor.arcsin()
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.arcsinh.md]: 这里应该使用`torch.Tensor.arcsin()`吧

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.select.md [hunk 12]
```diff
@@ -0,0 +1,22 @@
+## [ torch 参数更多 ]torch.Tensor.select
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.select.md]: 这里分类应该是`仅参数名不一致`吧

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.select.md [hunk 13]
```diff
@@ -0,0 +1,22 @@
...
+torch.Tensor.select(dim, index)
+```
+
+### [paddle.index_select](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/Tensor_cn.html#index-select-index-axis-0-name-none)
+
+```python
+paddle.Tensor.index_select(index, axis=0, name=None)
+```
+
+其中 Pytorch 相⽐ Paddle ⽀持更多其他参数，具体如下：
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.select.md]: 这句话的描述也不太对，应该是 `两者功能一致且参数用法一致，仅参数名不一致，具体如下：`，辛苦修改一下~

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.sigmoid.md [hunk 14]
```diff
@@ -1,24 +1,15 @@
...
+## [ 无参数 ]torch.Tensor.sigmoid
+
+### [torch.Tensor.sigmoid](https://pytorch.org/docs/stable/generated/torch.Tensor.sigmoid)
 
 ```python
 torch.Tensor.sigmoid()
 ```
 
-### [paddle.nn.functional.sigmoid](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/functional/sigmoid_cn.html)
+### [paddle.Tensor.sigmoid](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/Tensor/sigmoid_cn.html)
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.sigmoid.md]: 这个链接地址官网还没有更新，暂时为空就可以了~

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.slogdet.md [hunk 15]
```diff
@@ -0,0 +1,15 @@
+## [ 无参数 ]torch.Tensor.slogdet
+
+### [torch.Tensor.slogdet](https://pytorch.org/docs/stable/generated/torch.Tensor.slogdet.html?highlight=torch+tensor+slogdet#torch.Tensor.slogdet)
+
+```python
+torch.Tensor.slogdet()
+```
+
+### [paddle.Tensor.slogdet](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/Tensor_cn.html#slogdet-name-none)
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.slogdet.md]: 这个链接没有slogdet吧，直接为空就可以了~

**Comment by Fancy-hjyp** [comment]
同学，有时间的话可以根据修改意见修改一下，这样就能尽快合并了~

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.arcsinh.md [hunk 16]
```diff
@@ -0,0 +1,15 @@
+## [ 无参数 ]torch.Tensor.arcsinh
+
+### [torch.Tensor.arcsinh](https://pytorch.org/docs/stable/generated/torch.Tensor.arcsinh)
+
+```python
+torch.Tensor.arcsinh()
+```
+
+### [paddle.Tensor.asinh](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/Tensor_cn.html#asinh-name-none)
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.arcsinh.md]: 这个官网链接是空的吧，直接为空就可以了，目前在Tensor页面还没更新asinh~

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.asinh.md [hunk 17]
```diff
@@ -0,0 +1,15 @@
+## [ 无参数 ]torch.Tensor.asinh
+
+### [torch.Tensor.asinh](https://pytorch.org/docs/stable/generated/torch.Tensor.asinh)
+
+```python
+torch.Tensor.asinh()
+```
+
+### [paddle.Tensor.asinh](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/Tensor_cn.html#asinh-name-none)
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.asinh.md]: 这个链接也为空就可以~

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.slogdet.md [hunk 18]
```diff
@@ -0,0 +1,15 @@
+## [ 无参数 ]torch.Tensor.slogdet
+
+### [torch.Tensor.slogdet](https://pytorch.org/docs/stable/generated/torch.Tensor.slogdet.html?highlight=torch+tensor+slogdet#torch.Tensor.slogdet)
+
+```python
+torch.Tensor.slogdet()
+```
+
+### [paddle.Tensor.slogdet]()
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.slogdet.md]: 经过测试，目前还不支持直接在Tensor上使用该函数，只能通过paddle.linalg.slogdet进行调用。
所以这里分类需要改成仅paddle参数更多，调用的方法辛苦改成paddle.linalg.slogdet，一句话描述部分和参数映射部分也相应修改一下~，可以参考https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/linalg/slogdet_cn.html
```python
x =  paddle.randn([3,3,3])
y = paddle.linalg.slogdet(x)
print(y)
```

**Review by Fancy-hjyp — APPROVED** [review]
LGTM

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.slogdet.md [hunk 19]
```diff
@@ -0,0 +1,21 @@
+## [ 仅 paddle 参数更多 ]torch.Tensor.slogdet
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.slogdet.md]: 你好，这里辛苦改为 无参数 吧，根据最新的标准，这种情况不需要比对第一个参数了~

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.slogdet.md [hunk 20]
```diff
@@ -0,0 +1,21 @@
...
+torch.Tensor.slogdet()
+```
+
+### [paddle.linalg.slogdet](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/linalg/slogdet_cn.html)
+
+```python
+paddle.linalg.slogdet(x)
+```
+
+两者功能一致，其中 Paddle 相比 Pytorch 支持更多其他参数，具体如下：
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.slogdet.md]: 这里改为`两者功能一致，无参数`吧，下面的参数映射部分可以去掉

Comment (Fancy-hjyp) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.slogdet.md]: CI不通过的原因应该是服务器代理的问题，正好重新提交一下会触发CI
image

**Review by zhwesky2010 — APPROVED** [review]

**Comment by luotao1** [comment]
hi, @hongtushi 
- 非常感谢你对飞桨框架的贡献，我们正在运营一个PFCC组织，会通过定期分享技术知识与发布开发者主导任务的形式持续为飞桨框架做贡献，详情可见 https://github.com/luotao1 主页说明。
- 如果你对PFCC有兴趣，请发送邮件至 [ext_paddle_oss@baidu.com](mailto:ext_paddle_oss@baidu.com)，我们会邀请你加入~
